commands/welcome/help.js

```diff
@@ -61,33 +61,29 @@ module.exports = {
 
        Do \`${prefix}help <command>\` to see aliases of that command`)
        
-        .setColor("#FFFFF0")
-        .setFooter(`Code Owner: Atreya#2401`)
+      .setColor("#FFFFF0")
+      .setFooter(`Code Owner: Atreya#2401`)
     
 
-      let com = {};
-      for (let comm of commands.array()) {
-        let category = comm.category || "Unknown";
-        let name = comm.name;
+    const com = {};
+    for (let comm of commands.array()) {
+      const category = comm.category || "Unknown";
+      const { name } = comm;
```

Comment:
its comm.name

Reply:
You already declared variably `name`, so it would be more efficient that way. I hope you *test* the bot, because it's more cleaner code. If there's an *error* in the bot, then let me know :smile: